spec/ttml1.xml

```diff
@@ -4598,22 +4598,22 @@ as a style property only to those element types indicated in the following table
 computed value of the style property must be considered to be no less than the
 size of the largest font size that applies to the element and its
 descendant elements in the intermediate synchronic document as determined by
-<specref ref="semantics-region-layout-step-1"/>.</p>
+<specref ref="semantics-region-layout-step-1"/>. However, the value <code>normal</code> is not relative,
+and is therefore inherited as-is, as specified at <specref ref="semantics-style-resolution-processing-css"/>.</p>
 <note role="elaboration">
-<p>It is recommended that presentation processors use a computed line height that is
-125% of the size of the largest applicable font size when the value <code>normal</code>
-is specified. The choice of 125% allows use of a font size of <code>0.8c</code> in order
+<p>If the value of this attribute is <code>normal</code>, it is recommended that presentation processors
+use a line height that is 125% of the size of the largest applicable font size.
+The choice of 125% allows use of a font size of <code>0.8c</code> in order
 to obtain a line height of <code>1c</code>.</p>
 </note>
 <p>If specified as a <loc
 href="#style-value-length">&lt;length&gt;</loc>, then the length must
 be non-negative.</p>
-<p>If a computed value of the property associated with this attribute is not supported,
+<p>If the computed value of the property associated with this attribute is not supported,
 then a presentation processor must use the closest supported value.</p>
 <note role="elaboration">
 <p>In this context, the phrase <emph>closest supported value</emph> means the value for which the Euclidean distance between
-the computed line height and the supported line height is minimized. If there are multiple closest supported values equally distant from
```

Comment:
Other than source formatting (removal of a newline), there seems to be no change here. Please restore the newline in which case I believe this change is a no-op (unless there is something subtle I don't see here).

Reply:
It is unavoidable that whitespaces will be inserted in the editing process. One way to avoid this is to agree on a formatting tool that breaks lines consistently.

Reply:
No change should be made by the editing tool if there is no change in the source content. What is the change here that induced the diff delta? Also, no gratuitous (non-explicit) change in source whitespace should occur due to whatever editing tool is being used.